app/Models/InetCidrRoute.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+class InetCidrRoute extends DeviceRelatedModel
+{
+    protected $table = 'inetCidrRoute';
+//    protected $primaryKey = 'inetCidrRoute_id';
+    public $timestamps = true;
+
+    public function getPort()
```

Comment:
This should be defined as a port relationship, which means you probably want to have a port_id field on this table.